A66-otel-stats.md

```diff
@@ -0,0 +1,566 @@
+# OpenTelemetry Metrics
+
+*   Author: Yash Tibrewal (@yashykt)
+*   Approver: Mark Roth (@markdroth)
+*   Status: In Review
+*   Implemented in: <language, ...>
+*   Last updated: Jul 20, 2023
+*   Discussion at: https://groups.google.com/g/grpc-io/c/po-deqYEQzE
+
+## Abstract
+
+Propose a metrics data model for gRPC OpenTelemetry metrics.
+
+## Background
+
+There are a collection of
+[metrics](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/gRPC.md)
+proposed by OpenCensus for gRPC. OpenCensus is
+[no longer being actively maintained](https://opentelemetry.io/blog/2023/sunsetting-opencensus/),
+with OpenTelemetry suggested as the successor framework.
+
+### Related Proposals:
+
+*   [A6: gRPC Retry Design](A6-client-retries.md)
+*   [A39: xDS HTTP Filter Support](A39-xds-http-filters.md)
+*   [A45: Exposing OpenCensus Metrics and Tracing for gRPC retry](A45-retry-stats.md)
+
+## Proposal
+
+### Metrics Schema
```

Comment:
I think it might make sense to move this section down to line 412, right before the "Migration from OpenCensus" section.  Readers will probably care more about the APIs and architecture than about what metrics are being provided.

Reply:
moved